src/style.css

```diff
@@ -2,0 +1,29 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin: 0;
```

Comment:
Avoid setting margin to 0 for individual elements if you're going to set margins for specific cases anyway, like in `.stars__star:not(:first-child)`. This can be redundant and may lead to confusion when reading the code.